test-proxy/pom.xml

```diff
@@ -3,10 +3,11 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.google.cloud</groupId>
   <artifactId>google-cloud-bigtable-test-proxy</artifactId>
-  <version>1.0</version>
+  <version>0.0.1-SNAPSHOT</version>
   <packaging>jar</packaging>
   <name>Google Cloud Bigtable Test Proxy</name>
   <url>https://github.com/googleapis/java-bigtable</url>
+  <description>Cloud Bigtable Java Client test proxy for running conformance tests.</description>
 
   <parent>
     <artifactId>google-cloud-bigtable-parent</artifactId>
```

Comment:
Does the version tag below accept custom values (i.e., we want to test a version released a while ago)?